src/components/label/label.jsx

```diff
@@ -1,22 +1,22 @@
 import { h, Component } from 'preact';
-import {Localize} from '../../lib/localize';
+import { Localize } from '../../lib/localize';
 
 const lookup = new Localize().lookup;
 
 export default class Label extends Component {
 	static defaultProps = {
-		prefix: ''
+		altLocalizeKey: '',
+		prefix: '',
```

Comment:
Didn't you remove the trailing comma in your last PR?

Reply:
i dont think ive edited this file before, so it's just updating formatting as i go - so files have it, some dont, also the `{ spacing }` has been different across files as well lol. So that stuff is naturally getting reformatted since i dont care about upstream changes in the project anymore.